src/app/public/modules/phone-field/phone-field.component.html

```diff
@@ -0,0 +1,45 @@
+<div
+  class="sky-form-group sky-input-group"
+>
+  <div
+    class="sky-input-group-btn sky-input-group-country-btn"
```

Comment:
The class name `sky-input-group-country-btn` is a bit generic (see explanation below). Maybe use `sky-phone-field-country-btn` instead (or something similar)?

Reply:
Done